src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -153,6 +156,27 @@ public void onSpoutOpen(
 
         createSidelineTriggers();
 
+        final long refreshIntervalSeconds;
+
+        // This should be set by SidelineConfig.setDefaults(), but just in case we want to avoid the NullPointer
+        if (spoutConfig.containsKey(SidelineConfig.REFRESH_INTERVAL_SECONDS)) {
```

Comment:
I'd argue that since SpoutConfig or SidelineConfig or whatever sets a default, we should just add a precondition instead of having a second default.

Reply:
I'm fine with that.